misc/release/thentos-captcha-README.md

```diff
@@ -0,0 +1,15 @@
+# Thentos Captcha
```

Comment:
I propose to move this document into the `docs` folder (and adapt `thentos-captcha-release.sh` accordingly).